proto/momento/auth.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "momento.auth";
+
+package momento.auth;
+
+service Auth {
+  rpc Login (_LoginRequest) returns (stream _LoginResponse) {}
+}
+
+message _LoginRequest {
+}
```

Comment:
So this would be autheticated request with our JWT used to get sso login url? Just seems bit funny to me to have an authenticated request called _LoginRequest. Maybe more clear to call `SsoLogin`/ `_SsoLoginRequest`?

Reply:
> So this would be autheticated request with our JWT used to get sso login url?

No. This is untrusted callers asking to log in.

> Just seems bit funny to me to have an authenticated request called _LoginRequest.

It would be yes, but no - this is how unauthenticated users become authenticated users.

HMU on slack if you'd like to have a deeper conversation on the topic!

Reply:
Ok that makes sense to me then didn't know if this would be started totally unauthenticated from cli I think if thats case then this name is good 👍

Reply:
Yeah, today we don't have any unauthenticated -> authenticated API edge beyond an email. This will bring users in via normal identity provider idioms and establish a repeatable identity securely. It's going to be real neato.